app/screens/channel/channel_nav_bar/channel_drawer_button/index.js

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {connect} from 'react-redux';
+
+import {getUnreadsInCurrentTeam} from 'mattermost-redux/selectors/entities/channels';
+import {getCurrentTeamId, getTeamMemberships} from 'mattermost-redux/selectors/entities/teams';
+import {getTheme} from 'mattermost-redux/selectors/entities/preferences';
+
+import ChannelDrawerButton from './channel_drawer_button';
+
+function mapStateToProps(state) {
+    const currentTeamId = getCurrentTeamId(state);
+    const {mentionCount, messageCount} = getUnreadsInCurrentTeam(state);
+    const myTeamMembers = getTeamMemberships(state);
+
+    let mentions = mentionCount;
```

Comment:
Should this be refactored out into a selector since it's being recomputed frequently?